### ISSUE TITLE
linewrap auto doesn't work in jrnl v3

### ISSUE THREAD

**Issue by tjapro** [jrnl-org/jrnl#1688]
### Diagnostic output

jrnl: v3.0
Python: 3.11.1 (main, Jan  6 2023, 00:00:00) [GCC 12.2.1 20221121 (Red Hat 12.2.1-4)]
OS: Linux 6.1.12-200.fc37.x86_64

### Current Behavior

I was editing the `.config` file to other symbol for `indent_character`.  
Then when I run this `jrnl work -n 1`, I received this message.

```
┏━ Error ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
┃  TypeError                                                ┃
┃  '<=' not supported between instances of 'str' and 'int'  ┃
┃                                                           ┃
```

![image](https://user-images.githubusercontent.com/7900361/220777101-a2f391ce-5d06-416c-a61b-f4bd19cd545e.png)

![image](https://user-images.githubusercontent.com/7900361/220778226-b6819764-a3fa-4abf-8881-d1754807491f.png)

### Expected Behavior

No error.

### Repro Steps

I've provided.

### Debug output

```
jrnl --debug work -n 2                                    ~ 22:45:30 fsh
DEBUG    root         Parsed args: Namespace(debug=True, preconfig_cmd=None, postconfig_cmd=None, filename=None, on_date=None, today_in_history=False, month=None, day=None, year=None, start_date=None, end_date=None, contains=None, strict=False, starred=False, limit=2, excluded=[], edit=False, delete=False, change_time=None, export=False, tags=False, short=False, config_override=[], config_file_path='', text=['work'])
DEBUG    root         Reading configuration from file /home/tiagodalmeida/.config/jrnl/jrnl.yaml
DEBUG    root         Using configuration "{'colors': {'body': 'NONE', 'date': 'MAGENTA', 'tags': 'YELLOW', 'title': 'GREEN'}, 'default_hour': 10, 'default_minute': 0, 'editor': 'nano', 'encrypt': False, 'highlight': True, 'indent_character': '\ue621', 'journals': {'default': '~/Projects/jrnl/journal.txt', 'work': '~/Projects/jrnl/work.txt', 'story': '~/Projects/jrnl/story.txt'}, 'linewrap': 'auto', 'tagsymbols': '@', 'template': False, 'timeformat': '%a, %d %b %Y %H:%M:%S', 'version': 'v2.8.3'}"
DEBUG    root         Using journal name: work
DEBUG    root         opened PlainJournal with 1 entries
  File "/usr/lib/python3.11/site-packages/jrnl/cli.py", line 39, in cli
    status_code = run(args)
                  ^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/jrnl/jrnl.py", line 71, in run
    search_mode(**kwargs)
  File "/usr/lib/python3.11/site-packages/jrnl/jrnl.py", line 206, in search_mode
    _display_search_results(**kwargs)
  File "/usr/lib/python3.11/site-packages/jrnl/jrnl.py", line 410, in _display_search_results
    print(journal.pprint())
          ^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/jrnl/Journal.py", line 183, in pprint
    return "\n".join([e.pprint(short=short) for e in self.entries])
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/jrnl/Journal.py", line 183, in <listcomp>
    return "\n".join([e.pprint(short=short) for e in self.entries])
                      ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/jrnl/Entry.py", line 108, in pprint
    title = ansiwrap.fill(
            ^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/ansiwrap/core.py", line 105, in fill
    return '\n'.join(wrap(s, width, **kwargs))
                     ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/ansiwrap/core.py", line 91, in wrap
    wrapped = a_textwrap.wrap(s, width, **kwargs)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/textwrap3.py", line 427, in wrap
    return w.wrap(text)
           ^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/textwrap3.py", line 402, in wrap
    return self._wrap_chunks(chunks)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/textwrap3.py", line 295, in _wrap_chunks
    if self.width <= 0:
       ^^^^^^^^^^^^^^^
╭──────────────────── Traceback (most recent call last) ─────────────────────╮
│ /usr/lib/python3.11/site-packages/jrnl/cli.py:39 in cli                    │
│                                                                            │
│   38 │   │                                                                 │
│ ❱ 39 │   │   status_code = run(args)                                       │
│   40                                                                       │
│                                                                            │
│ /usr/lib/python3.11/site-packages/jrnl/jrnl.py:71 in run                   │
│                                                                            │
│    70 │   else:                                                            │
│ ❱  71 │   │   search_mode(**kwargs)                                        │
│    72                                                                      │
│                                                                            │
│ /usr/lib/python3.11/site-packages/jrnl/jrnl.py:206 in search_mode          │
│                                                                            │
│   205 │   else:                                                            │
│ ❱ 206 │   │   _display_search_results(**kwargs)                            │
│   207                                                                      │
│                                                                            │
│ /usr/lib/python3.11/site-packages/jrnl/jrnl.py:410 in                      │
│ _display_search_results                                                    │
│                                                                            │
│   409 │   else:                                                            │
│ ❱ 410 │   │   print(journal.pprint())                                      │
│   411                                                                      │
│                                                                            │
│ /usr/lib/python3.11/site-packages/jrnl/Journal.py:183 in pprint            │
│                                                                            │
│   182 │   │   """Prettyprints the journal's entries"""                     │
│ ❱ 183 │   │   return "\n".join([e.pprint(short=short) for e in self.entrie │
│   184                                                                      │
│                                                                            │
│ /usr/lib/python3.11/site-packages/jrnl/Journal.py:183 in <listcomp>        │
│                                                                            │
│   182 │   │   """Prettyprints the journal's entries"""                     │
│ ❱ 183 │   │   return "\n".join([e.pprint(short=short) for e in self.entrie │
│   184                                                                      │
│                                                                            │
│ /usr/lib/python3.11/site-packages/jrnl/Entry.py:108 in pprint              │
│                                                                            │
│   107 │   │   │   # Color date / title and bold title                      │
│ ❱ 108 │   │   │   title = ansiwrap.fill(                                   │
│   109 │   │   │   │   date_str                                             │
│                                                                            │
│ /usr/lib/python3.11/site-packages/ansiwrap/core.py:105 in fill             │
│                                                                            │
│   104 │   """                                                              │
│ ❱ 105 │   return '\n'.join(wrap(s, width, **kwargs))                       │
│   106                                                                      │
│                                                                            │
│ /usr/lib/python3.11/site-packages/ansiwrap/core.py:91 in wrap              │
│                                                                            │
│    90 │   kwargs = _unified_indent(kwargs)                                 │
│ ❱  91 │   wrapped = a_textwrap.wrap(s, width, **kwargs)                    │
│    92 │   return ansi_terminate_lines(wrapped)                             │
│                                                                            │
│ /usr/lib/python3.11/site-packages/textwrap3.py:427 in wrap                 │
│                                                                            │
│   426 │   w = TextWrapper(width=width, **kwargs)                           │
│ ❱ 427 │   return w.wrap(text)                                              │
│   428                                                                      │
│                                                                            │
│ /usr/lib/python3.11/site-packages/textwrap3.py:402 in wrap                 │
│                                                                            │
│   401 │   │   │   self._fix_sentence_endings(chunks)                       │
│ ❱ 402 │   │   return self._wrap_chunks(chunks)                             │
│   403                                                                      │
│                                                                            │
│ /usr/lib/python3.11/site-packages/textwrap3.py:295 in _wrap_chunks         │
│                                                                            │
│   294 │   │   lines = []                                                   │
│ ❱ 295 │   │   if self.width <= 0:                                          │
│   296 │   │   │   raise ValueError("invalid width %r (must be > 0)" % self │
╰────────────────────────────────────────────────────────────────────────────╯
TypeError: '<=' not supported between instances of 'str' and 'int'
┏━ Error ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
┃  TypeError                                                ┃
┃  '<=' not supported between instances of 'str' and 'int'  ┃
┃                                                           ┃
┃  This is probably a bug. Please file an issue at:         ┃
┃  https://github.com/jrnl-org/jrnl/issues/new/choose       ┃
┗━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┛

```

**Comment by micahellison** [comment]
Hello @tjapro, thank you for the bug report.

I'm not able to reproduce this issue, though I'm not sure if it's because I'm running a more recent jrnl version, have a different OS, or a different locale, or something else.

Here's my diagnostic
```
jrnl: v3.3
Python: 3.11.1 (tags/v3.11.1:a7a450f, Dec  6 2022, 19:58:39) [MSC v.1934 64 bit (AMD64)]
OS: Windows 10
```

Could you try upgrading to jrnl v3.3 then letting us know if the issue persists?

**Comment by wren** [comment]
Hi, @tjapro. Thanks for filing this issue.

It looks like you're trying to use `linewrap: auto` on jrnl v3.0. As you can see from our changelog, this feature was implemented in jrnl v3.1. So, please update jrnl, or change the value of `linewrap` and you should stop seeing this error.

**Comment by wren** [comment]
Also, for the value of `indent_character`, you want to paste actual character into your config file, not the unicode character for it (otherwise you'll see the code instead of the character).

So, for example:

Correct:
```yaml
indent_character: '│'
```

Incorrect:
```yaml
indent_character: '\u2502'
```

**Comment by tjapro** [comment]
> Hi, @tjapro. Thanks for filing this issue.
> 
> It looks like you're trying to use `linewrap: auto` on jrnl v3.0. As you can see from our changelog, this feature was implemented in jrnl v3.1. So, please update jrnl, or change the value of `linewrap` and you should stop seeing this error.

Yup, that worked out. And also, thanks for the attention to the unicode vs glyph.  
Maybe I'll add this to the docs ...
